apps/dashboard/src/contract-ui/tabs/settings/components/metadata.tsx

```diff
@@ -111,20 +107,10 @@ export const SettingsMetadata = <
     name: "dashboard_social_urls",
   });
 
-  const chain = useV5DashboardChain(contract?.chainId);
-  const contractV5 =
-    contract && chain
-      ? getContract({
-          address: contract.getAddress(),
-          chain: chain,
-          client: thirdwebClient,
-        })
-      : null;
-
   const { onSuccess, onError } = useTxNotifications(
     "Successfully updated metadata",
     "Error updating metadata",
-    contractV5,
+    contract,
   );
```

Comment:
not a blocker but I would like to request an update to this hook (and it also no longer has to be a hook in this world):

we now do a lot of `await mutation.mutateAsync()` -> can we please make a new version of this that takes a `mutation.mutateAsync` promise result (from the tx hook) -> passes it to sonner for toasts etc?

Reply:
I made the linear ticket (CNCT-1163)
will work on it